data/lang/core/en.json

```diff
@@ -887,6 +887,10 @@
     "description": "",
     "message": "missile"
   },
+  "MISSILE_DETECTED": {
+    "description": "",
+    "message": "Missile launch detected."
+  },
```

Comment:
Just a thought: will this trigger for any missile launch (except for the player's, of course?), or just for missiles aimed at the player?

Reply:
(I'm asking to see if "Incomming missile detected" is apt, but would only work if this is unique to when only the AI targets the Player.)

Reply:
It is triggered for any missile launch similar "Ship firing"